frontend/packages/console-app/locales/en/console-app.json

```
@@ -224,11 +224,14 @@
   "{{section}} not found": "{{section}} not found",
   "Enabled": "Enabled",
   "Disabled": "Disabled",
+  "This plugin seems to have violated Console Content Security Policy. Refer to the browser's console logs for details.": "This plugin seems to have violated Console Content Security Policy. Refer to the browser's console logs for details.",
```

What if we say something like 
```suggestion
  "This plugin might have violated Console Content Security Policy. Refer to the browser's console logs for details.": "This plugin might have violated Console Content Security Policy. Refer to the browser's console logs for details.",
```
I can't find anything anywhere stating not to use "seems" but it feels indirect. If this stems from the console being unable to tell if a CSP violation originated from a console plugin then I think "might" would make more sense.

---

@vojtechszocs, It looks like you lost @opayne1's suggestion when rebasing.